Formula/avro-c.rb

```diff
@@ -26,8 +26,10 @@ class AvroC < Formula
   uses_from_macos "zlib"
 
   def install
-    system "cmake", ".", *std_cmake_args
-    system "make", "install"
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
     pkgshare.install "tests/test_avro_1087"
```

Comment:
```
Error: An exception occurred within a child process:
    Errno::ENOENT: No such file or directory - tests/test_avro_1087
```

Build may be skipping compiling test now. May want to just install a C test code and compile it during the test block.